src/core/thread/discover_scanner.hpp

```diff
@@ -109,6 +112,7 @@ class DiscoverScanner : public InstanceLocator
                      Mac::PanId              aPanId,
                      bool                    aJoiner,
                      bool                    aEnableFiltering,
+                     const Mac::ExtAddress * aFilterId,
```

Comment:
Should we allow for something more general? Either a `uint64_t` or even a byte array?

Reply:
I actually changed this in new pushed commit to use bloom filter (steering data) `FilterIndexes` directly.